codebases/signalen.md

```diff
@@ -0,0 +1,53 @@
+# Signalen
+
+> Stage 2: assessment
+
+## What it is
+
+Signalen makes it easier for residents to make a nuisance report, and helps the city respond to the most important things first.
+
+It includes:
+
+* algorithmic processing by type of complaint and second layer triaging and prioritization by humans
+* automatic updates to complainants about the status of their report
+* automatic alerts to city staff based on analysis of combined reports
+* easy report transfer to other city workers and subcontractors
+
+## How we're helping
+
+We’re helping Amsterdam and the VNG define a governance structure, including shared feature roadmaps and technical steering.
+
+We’re also helping make the codebase more conducive to collaboration, including:
+
+* clear branching and gitflow models
+* automated unit tests and integration testing
+* engineering guidelines
+* help prepare the community to empower market parties to offer the solution as SaaS
```

Comment:
Should we have SaaS as an abbreviation?

Reply:
Good spot - I've changed this setence now.